src/index.js

```diff
@@ -0,0 +1,76 @@
+import './index.css';
+import Task from '../modules/newTask.js';
+import addtask from '../modules/addTask.js';
+import removetask from '../modules/removeTask.js';
+import edittask from '../modules/editTask.js';
+import completed from '../modules/complete.js';
+
+const todoList = document.getElementById('todoList');
+
+// displaying tasks on window loading
+window.addEventListener('DOMContentLoaded', () => {
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+
+  if (tasks !== null) {
+    tasks.sort((a, b) => a.index - b.index);
+
+    tasks.forEach((item) => {
+      todoList.innerHTML += `
+        <div class="todoFlex">
+          <div class="todoDiv">
+            <input type="checkbox" name="${item.id}" class="checkbox" ${item.completed ? 'checked' : ''} >
+            <input class="todoP" name="${item.id}" value="${item.description}" >
+          </div>
+          <i class="bi bi-trash3-fill dots" id="${item.id}" onclick="remove(this)"></i>
+        </div>
+      `;
```

Comment:
Please it my suggestion to use appendChild method rather than innerHTML for project security purposes